[backport to 3.5] bpo-28929: Link the documentation to its source file on GitHub

Mariatta opened the pull request:
Change the documentation's `Show Source` link on the left menu
to GitHub source file.

(cherry picked from commit 23bafa294c75c20cb85ae5d97d7571a3a0ad8dd3)

AnishShah commented:
Sorry.
Created this PR by mistake. I was trying to create PR on my repo itself.
I apologize.

Mariatta committed:
bpo-28929: Link the documentation to its source file on GitHub

Change the documentation's `Show Source` link on the left menu
to GitHub source file.

(cherry picked from commit 23bafa294c75c20cb85ae5d97d7571a3a0ad8dd3)

Mariatta committed:
remove if statement
